src/api.js

```diff
@@ -417,7 +412,7 @@ var remoteAuthentication = function(options, entry) {
           'sha256',
           req.headers['content-type']
         );
-        if (!cryptiles.fixedTimeComparison(result.hash, hash)) {
+        if (!crypto.timingSafeEqual(result.hash, hash)) {
```

Comment:
nice spot

Reply:
I would love to take credit, but really what happened is that cryptiles removed `fixedTimeComparison` from their most recent version since `timingSafeEqual` was added to the std lib.